Trumpf.Coparoo.Web/Controls/Table/TableBody.cs

```diff
@@ -1,34 +1,15 @@
 ﻿namespace Trumpf.Coparoo.Web.Controls.Table
 {
     using OpenQA.Selenium;
-    using System.Collections.Generic;
-    using System.Linq;
 
     /// <summary>
     /// Table body control object.
     /// </summary>
-    public class TableBody : ControlObject
+    public class TableBody : TableSegment
```

Comment:
What about shortending the name to simply `Body`? This is still unique in the namespace.
Same for `TableCell`, `TableFoot`, `TableHead` and `TableSegment `?